deps/config/manifests/roles.yml.template

```diff
@@ -14,7 +14,7 @@
     permission:
       - "read:*"
       - "sign:keys"
-      - "sign:eth1accounts"
+      - "sign:ethereum"
```

Comment:
I think this should be `sign:ethaccount`

Reply:
why? `ethereum` is clearly a better name than `ethaccounts` since the domain is named `ethereum` so it correlates with keys and secrets